catalog/models.py

```diff
@@ -1,3 +1,40 @@
 from django.db import models
 
-# Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(max_length=50, verbose_name="Наименование")
+    description = models.TextField(null=True, blank=True, verbose_name="Описание")
+
+    def __str__(self):
+        return f"{self.name}"
```

Comment:
name - это строка, поэтому форматировать не нужно
```suggestion
        return self.name
```